src/sqlparser/src/keywords.rs

```diff
@@ -763,7 +763,7 @@ pub const RESERVED_FOR_COLUMN_OR_TABLE_NAME: &[Keyword] = &[
     Keyword::WHEN,
     Keyword::WHERE,
     Keyword::WINDOW,
-    // Keyword::WITH,
+    Keyword::WITH,
```

Comment:
I just found on the comment on this function that there was a reason why WITH is specially handled. Can you test that?

https://github.com/chagelo/risingwave/blob/6bde5e2fe2c64dfcb3777eae9271c97aef3e2b4b/src/sqlparser/src/keywords.rs#L662-L663

Reply:
Not sure what's "won't parse".